### PR TITLE
[chores] Pin version for dependencies

### PR DESCRIPTION
**Pull request by nepython**
Checks:

- [ ] I have manually tested the proposed changes
- [ ] I have written new test cases to avoid regressions (if necessary)
- [ ] I have updated the documentation (e.g. README.rst)

Refer to https://github.com/openwisp/openwisp-monitoring/pull/227#discussion_r481200229
@nemesisdesign, do you think we should also pin down test dependencies?

### CODE REVIEW

**Comment by nepython** [comment]
For some reason influxdb-5.3 is getting installed :thinking:
https://travis-ci.org/github/openwisp/openwisp-monitoring/jobs/723119226#L613-L614
Something seems incorrect with this,
```shell
influxdb~=5.2
```
I expected the latest version in 5.2.x to be installed

**Comment by nemesisdesign** [comment]
> For some reason influxdb-5.3 is getting installed 
> https://travis-ci.org/github/openwisp/openwisp-monitoring/jobs/723119226#L613-L614
> Something seems incorrect with this,
> 
> ```shell
> influxdb~=5.2
> ```
> 
> I expected the latest version in 5.2.x to be installed

5.3 is the latest one: https://github.com/influxdata/influxdb-python/releases/tag/v5.3.0

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 openwisp-controller~=0.7.post1
-influxdb>=5.2,<5.3
-django-notifications-hq>=1.6,<1.7
-django-celery-email>=3.0.0,<3.1
-djangorestframework>=3.11,<3.12
+influxdb~=5.2
+django-notifications-hq~=1.6
+django-celery-email~=3.0.0
+djangorestframework~=3.11
```
Comment (nemesisdesign) [inline requirements.txt:5]: I also noticed this now. We can rely on `openwisp-utils[rest]`, which we made on purpose to centralize the DRF version there (so we can update it there to update it on all the modules).
I think we can leave `openwisp-utils[rest]` unpinned so the version used is the one dictated by `openwisp-controller`. Can you try that please?

**Comment by nepython** [comment]
> > For some reason influxdb-5.3 is getting installed
> > https://travis-ci.org/github/openwisp/openwisp-monitoring/jobs/723119226#L613-L614
> > Something seems incorrect with this,
> > ```shell
> > influxdb~=5.2
> > ```
> > 
> > 
> > I expected the latest version in 5.2.x to be installed
> 
> 5.3 is the latest one: https://github.com/influxdata/influxdb-python/releases/tag/v5.3.0

I think then we can pin `influxdb` version in #227 that can be merged after this PR (since it is dedicated for it).

File: requirements.txt [hunk 2]
```diff
@@ -1,8 +1,8 @@
 openwisp-controller~=0.7.post1
+openwisp-utils[rest]
 influxdb>=5.2,<5.3
-django-notifications-hq>=1.6,<1.7
-django-celery-email>=3.0.0,<3.1
-djangorestframework>=3.11,<3.12
+django-notifications-hq~=1.6
```
Comment (nemesisdesign) [inline requirements.txt:4]: let's replace this with `openwisp-notifications~=0.1`, which has been released, related to https://github.com/openwisp/openwisp-monitoring/issues/231. 

I would not remove it from install-dev yet.

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (nepython) [reply requirements.txt]: I haven't tagged the issue (as to be closed) as we are not yet removing the openwisp-notifications `dev` (which makes sense given the addition of many new changes).

File: requirements.txt [hunk 3]
```diff
@@ -1,8 +1,8 @@
 openwisp-controller~=0.7.post1
+openwisp-utils[rest]
```
Comment (nemesisdesign) [inline requirements.txt:2]: Always pin dependencies, I could do it myself but I want you to do it so you won't forget next time, thanks.

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 3 above)
Comment (nepython) [reply requirements.txt:2]: @nemesisdesign, you explicitly mentioned to leave it unpinned above https://github.com/openwisp/openwisp-monitoring/pull/228#discussion_r481320603 and your reason for doing so seemed good enough to me.

Comment (nemesisdesign) [reply requirements.txt:2]: LOL! you're right @nepython, but I forgot already. Let's add a comment to avoid forgetting, can you please add one?

**Review by nemesisdesign — APPROVED** [review]
:+1:

**Pull request merged**